go.mod

```diff
@@ -23,6 +24,8 @@ require (
 	github.com/opencontainers/image-spec v1.0.2
 	github.com/pkg/errors v0.9.1
 	github.com/shteou/go-ignore v0.3.0
+	github.com/sigstore/cosign v1.3.1
+	github.com/sirupsen/logrus v1.8.1
```

Comment:
While I'm no conftest maintainer, it seems rather odd to introduce an entire logging system for use in only two packages. Since conftest does not run as a server, I'm not sure a logging library would make sense even if introduced consistently in all packages, or WDYT?

Reply:
thanks, I removed the logrus modules for logging.